paper-ieee-trans.tex

```diff
@@ -339,7 +340,7 @@ \subsection{INtegrating Distributed data Infrastructures for Global
 project of software development considered in this paper. At the time of writing
 this paper, the project is succeeding in providing solutions to address the existing gaps in
 Cloud Platform--as--a--Service (PaaS) and Software--as--a--Service (SaaS) levels,
-helping developers, e--Infrastructures and scientific communities to exploit
+helping developers, e--Infrastructures administrators and scientific communities to exploit
```

Comment:
e-Infrastructure administrators

Reply:
agree

Reply:
ok

Reply:
ok

Reply:
ok ( the "--" I've put from time to time, by mistake :))